src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java

```diff
@@ -115,20 +115,18 @@ public void sessionCreated(IoSession session) throws Exception {
 
         session.getFilterChain().addFirst("proxy", new ProxyFilter(isClient ? "client" : "server"));
 
-        if (true) {
+        String fileName = System.currentTimeMillis() + '_' + forward.getHostName() + '_' + forward.getPort() + '_' + (isClient ? "DOWNSTREAM" : "UPSTREAM");
 
-            String fileName = System.currentTimeMillis() + '_' + forward.getHostName() + '_' + forward.getPort() + '_' + (isClient ? "DOWNSTREAM" : "UPSTREAM");
+        File headersFile = loader.getResource(dumpTo + fileName + ".cap").getFile();
+        headersFile.createNewFile();
 
-            File headersFile = loader.getResource(dumpTo + fileName + ".cap").getFile();
-            headersFile.createNewFile();
+        File rawFile = loader.getResource(dumpTo + fileName + ".raw").getFile();
+        rawFile.createNewFile();
 
-            File rawFile = loader.getResource(dumpTo + fileName + ".raw").getFile();
-            rawFile.createNewFile();
-
-            FileOutputStream headersFos = new FileOutputStream(headersFile);
+        FileOutputStream headersFos = new FileOutputStream(headersFile);
+        FileOutputStream rawFos = new FileOutputStream(rawFile);
+        try {

```

Comment:
I bumped the pom's up-to 1.7, but they aren't committed yet; I don't want to push 1.8 just yet even though that's what everyone should be on by this point based on the JDK end-of-line dates.